src/extrapolation/extrapolation.jl

```diff
@@ -28,7 +31,7 @@ You can also combine schemes in tuples. For example, the scheme `(Linear(), Flat
 Finally, you can specify different extrapolation behavior in different direction. `((Linear(),Flat()), Flat())` will extrapolate linearly in the first dimension if the index is too small, but use constant etrapolation if it is too large, and always use constant extrapolation in the second dimension.
 """
 extrapolate{T,N,IT,GT,ET<:ExtrapDimSpec}(itp::AbstractInterpolation{T,N,IT,GT}, et::ET) =
-    Extrapolation(T,N,itp,IT,GT,et)
+    Extrapolation(T,Val{N},itp,IT,GT,et)
```

Comment:
Seems better to call `Extrapolation{T,N,ITPT,IT,GT,ET}(itp)` directly. We should probably deprecate the method on line 4.